site/src/atomics/colors.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Color
+description: Color and background-color related atomic css classes for the Atlas Design System
+template: standard
+classType: Atomics
+classPrefixes:
+  - color
+  - background-color
+---
+
+# Color Atomics
+
+Atlas has CSS classes that allow you to easily apply themed background and text colors. Change the site's theme to see how each looks in different themes.
+
+## Accessibility concerns
+
+Atlas components ensure accessible text-to-background color contrast ratios. However, when applying specific colors yourself via atomics, always remember to test contrast.
+
+## Text colors
+
+Atlas has three themed text colors.
+
+<table class="table">
+	<caption>
+		Text color classes
+	</caption>
+	<thead>
+		<tr>
+			<th>Text color</th>
+			<th>Text class</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr class="color-text">
+			<td>Text</td>
+			<td><code>.color-text</code></td>
+		</tr>
+		<tr class="color-text-subtle">
+			<td>Text subtle</td>
+			<td><code>.color-text-subtle</code></td>
+		</tr>
+		<tr class="color-text-invert background-color-alternate">
+			<td>Text invert</td>
+			<td><code>.color-text-invert</code></td>
+		</tr>
+	</tbody>
+</table>
+
+## Background colors
+
+Atlas has five themed (non-semantic) background colors.
+
+<table class="table">
+	<caption>
+		Body color classes
+	</caption>
+	<thead>
+		<tr>
+			<th>Background color</th>
+			<th>Background class</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr class="background-color-body">
+			<td>Body</td>
+			<td><code>.background-color-body</code></td>
+		</tr>
+		<tr class="background-color-body-medium">
+			<td>Body medium</td>
+			<td><code>.background-color-body-medium</code></td>
+		</tr>
+		<tr class="background-color-body-accent">
+			<td>Body accent</td>
+			<td><code>.background-color-body-accent</code></td>
+		</tr>
+		<tr class="background-color-alternate color-text-invert">
+			<td>Alternate</td>
+			<td><code>.background-color-alternate</code></td>
+		</tr>
+		<tr class="background-color-alternate-medium color-text-invert">
+			<td>Alternate medium </td>
+			<td><code>.background-color-alternate-medium</code></td>
+		</tr>
+	</tbody>
+</table>
+
+## Semantic colors
+
+Semantic colors are themed colors that vary depending on the active theme. They have names that pertain to their usage. Generally speaking, these are the colors used to paint components too.
+
+<table class="table">
+	<caption>
+		Primary color classes
+	</caption>
+	<thead>
+		<tr>
+			<th>Background color /<br>Text color</th>
+			<th>Font class</th>
+			<th>Background class</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr class="background-color-primary-invert color-primary">
+			<td>Primary invert /<br>Primary</td>
+			<td><code>.color-primary</code></td>
+			<td><code>.background-color-primary-invert</code></td>
+		</tr>
+		<tr class="background-color-primary-light" style="color: var(--theme-primary-dark)">
```

Comment:
The style attribute tells me that we may want to make `color-${themed-color}-dark` available (which I can add in upcoming PRs).

Reply:
I don't know if we do, to be honest. I think it's better used with the CSS as component level backgrounds, rather than any other more customized way.